README.md

```diff
@@ -92,16 +77,21 @@ Both the indexer and the web service allow for project-specific customizations
 via a plug-in mechanism, allowing the Boardwalk UI codebase to be functionally
 generic with minimal need for project-specific behavior.
 
-# 1. Getting Started
+## 1.2 Architecture Diagram
 
-## 1.1 Development Prerequisites
+![Azul architecture diagram](docs/azul-arch.png)
 
-- Python 3.6 (3.7 does not work) with `pip`
+# 2. Getting Started
+
+## 2.1 Development Prerequisites
+
+- Python 3.6 (3.7 does not work) with `pip`. Micro versions 3.6.5 and 3.6.8 are 
+  known to work, but 3.6.9 does not.
 
 - The `bash` shell
 
 - Docker for running the tests. The minimal required version is uncertain but
-  18.09 and 17.09 are known to work
+  19.03, 18.09, and 17.09 are known to work.
```

Comment:
As we discovered today, you seem to have issues with `make format`. Let's see if those have anything to do with your use of 19.03.  Then again I'm using 19.03 as well and `make format` seems to work.

Reply:
I downgraded to 18.09 and the error persists.